src/main.py

```diff
@@ -1,18 +1,14 @@
-"""The entry point of the application."""
 
 import sys
+import os
 
-from src.example import greeting
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..', 'src')))
 
+from main_application import MainApplication
```

Comment:
Nem kell piszkálni a sys.path-t, hanem relatív importot kell használni és `python -m src.main` paranccsal indítani.
```suggestion
from .main_application import MainApplication
```